diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/BracesInConditionalsAndLoopsRule.kt

```diff
@@ -152,6 +145,20 @@ class BracesInConditionalsAndLoopsRule(configRules: List<RulesConfig>) : DiktatR
         }
     }
 
+    private fun ASTNode.findIndentBeforeNode(): Int {
+        val indentNode = if (treeParent?.treeParent?.treeParent?.elementType == LAMBDA_EXPRESSION) {
+            treeParent.prevSibling { it.elementType == WHITE_SPACE }
+        } else {
+            prevSibling { it.elementType == WHITE_SPACE }
```

Comment:
```suggestion
            prevSibling { it.elementType == WHITE_SPACE } ?: throw IllegalStateException("Not found WHITE_SPACE in prev siblings")
```

Reply:
Not necessary, because safe null call added for case when `if` comes after bracket.